micronota/db/uniref.py

```diff
@@ -262,10 +171,8 @@ def sort_uniref(db_fp, uniref_fp, out_d, overwrite=False):
     out_d : str
         The output directory.
     '''
-    name_map = {'Swiss-Prot': 'sprot',
-                'TrEMBL': 'trembl'}
-    fns = ['%s_%s' % (i, j) for i in ['sprot', 'trembl']
-           for j in ['Bacteria', 'Archaea', 'Viruses', 'other']]
+    fns = ['%s_%s' % (i, j) for i in _status
```

Comment:
:+1:  it is ok just leave it as it is not so cluttered.

On Fri, Feb 19, 2016 at 2:31 PM, Daniel McDonald <notifications@github.com>
wrote:

> In micronota/db/uniref.py
> <https://github.com/biocore/micronota/pull/44#discussion_r53528236>:
>
> > @@ -262,10 +171,8 @@ def sort_uniref(db_fp, uniref_fp, out_d, overwrite=False):
> >      out_d : str
> >          The output directory.
> >      '''
> > -    name_map = {'Swiss-Prot': 'sprot',
> > -                'TrEMBL': 'trembl'}
> > -    fns = ['%s_%s' % (i, j) for i in ['sprot', 'trembl']
> > -           for j in ['Bacteria', 'Archaea', 'Viruses', 'other']]
> > +    fns = ['%s_%s' % (i, j) for i in _status
>
> itertools.product(_status, _kingdom)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/micronota/pull/44/files#r53528236>.
>